src/components/ebay-combobox/test/test.browser.js

```diff
@@ -378,7 +378,7 @@ describe('given the combobox starts with zero options', () => {
         });
 
         it('then it should not be expanded', () => {
-            expect(component.getByRole('combobox')).not.has.attr('aria-expanded');
+            expect(component.getByRole('combobox')).has.attr('aria-expanded', 'false');
```

Comment:
@ianmcburnie is it mandatory to not to have this attribute on load?

Reply:
I'd assumed absence of attribute on a combobox would be applied as aria-expanded=false. It definitely doesn't hurt to be explicit though and have it present in all cases, especially if that is what axe is looking for.